packages/frontend-2/components/settings/workspaces/General/DeleteDialog.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+  <LayoutDialog
+    v-model:open="isOpen"
+    title="Delete workspace"
+    max-width="sm"
+    :buttons="dialogButtons"
+  >
+    <p>
+      Are you sure you want to
+      <span class="font-medium">permanently delete</span>
+      the selected workspace?
+    </p>
+    <div
+      class="rounded border bg-foundation-2 border-outline-3 text-body-2xs text-foreground font-medium py-3 px-4 my-4"
+    >
+      {{ workspace.name }}
+    </div>
+    <p>
+      This action
+      <span class="font-medium">cannot</span>
```

Comment:
I'd still add the "type in your workspace name" check, even if there's no official designs and we have to go by the same design we have in project settings. This just feels way too easy to delete

Reply:
I agree but Michal was thinking about various options for the check, it was previously discussed to leave it out for now and implement it properly once he is back